### PR TITLE
etc: updated ufw firewall application preset

### PR DESCRIPTION
**Pull request by Stefan-Code**
Updated the ufw firewall profile to include the syncthing gui port (optionally).

### CODE REVIEW

File: etc/firewall-ufw/syncthing [hunk 1]
```diff
@@ -2,3 +2,8 @@
 title=Syncthing
 description=Syncthing file synchronisation
 ports=22000/tcp|21027/udp
+
+[syncthing-gui]
+title=Syncthing-gui
```
Comment (calmh) [inline etc/firewall-ufw/syncthing]: Capitalization of GUI here and I'm happy :)

Comment (Stefan-Code) [inline etc/firewall-ufw/syncthing]: Fixed it.

**Comment by Stefan-Code** [comment]
I rebased to include the latest changes to syncthing/master.

**Comment by calmh** [comment]
@st-review merge

etc: Updated ufw firewall application preset with default GUI port

**Comment by st-review** [comment]
:ok_hand: Merged as 7157917a1638a3ced9ec0c2feb3fbf95800a901a. Thanks, @Stefan-Code!

**Pull request closed without merging**